cdc/cdc/kv/region_worker.go

```diff
@@ -730,31 +730,39 @@ func (w *regionWorker) handleResolvedTs(
 		return nil
 	}
 	regionID := state.sri.verID.GetID()
+
+	safeResolvedTs := GetSafeResolvedTs(resolvedTs)
+	if safeResolvedTs == 0 {
```

Comment:
If we accept a `resolvedTs` less that safe interval, suggest to move the warning into `GetSafeResolvedTs`.